source/help/getting-started/signing-in.rst

```diff
@@ -118,8 +118,11 @@ How Mattermost determines your status
     :header: "Client", "**Online**", "**Away**", "**Offline**"
 
     "**Desktop App**", "You're interacting with your computer", "You're inactive on your computer for 5 minutes", "You close Mattermost, sleep or lock your computer"
```

Comment:
You're inactive on your computer for 5 minutes > You have no mouse movement on your computer for 5 minutes

Reply:
@esethna could you please explain further , do you mean the second sentence is alot better?

Reply:
Following up on this, I think @esethna's suggestion may work in terms of clarity. I propose this wording: "You haven't moved your mouse for more than 5 minutes". Let me know what you think though, I am 1/5 on it.

Reply:
Well, I am not entirely sure because someone could be on another app window and not necessarily moving your mouse I don't know if that counts

Reply:
That's a very good point. From my observations I have been marked as "away" when I've been in another window with Mattermost open in the background. I'm going to test that out today and revert.